### DISCUSSION TITLE
AWS EBS Volume space keeps increasing after the EC2 instance gets terminated.

### DISCUSSION THREAD

**Discussion by Priya-Devops2023** [learndevopsonline/learndevopsonline#1818] [Q&A]
Hi,
   My EBS volume space is keep increasing day by day and reached Maximum So I terminated the EC2 instances but still EBS volume space is increasing.Cant understand whats happening ? I am using AWS free tier now my EBS volume space has reached to 94% 28 GB has been used.

I have attached my aws account screenshot.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/154531102/d8ad9489-b00e-42b3-a2b1-508a36fc593b)
![EBS volume space problem](https://github.com/learndevopsonline/learndevopsonline/assets/154531102/7b781548-06b5-4abf-bd2d-a9b4053a81f0)

Kindly let me know if anyone has any solution for this.Thanks in advance.

Thanks and Regards
Priya

**Comment by chaithanya1812** [comment]
Hello Priya 
No need to worry, EBS storage doesn't generate bill to much, Still we can request AWS to wave off the bill.

**Reply by Priya-Devops2023** [reply]
Thanks. so no solution for clearing the space ??

**Reply by chaithanya1812** [reply]
Solution means you need terminate all instances in all regions, but don't do this.

**Reply by Priya-Devops2023** [reply]
I checked all the regions no instances are running in any region.For now i will park this problem.Thanks for the help Chaithanya.

**Reply by chaithanya1812** [reply]
Hello Priya, Actually  it's showing this month percentage of EBS you have been used.
It will be there upto this month end. next month it starts from 0% again. So no need to worry about this.
